boards/sparc/bm3803/xx3803/scripts/Make.defs

```diff
@@ -24,19 +24,8 @@ include ${TOPDIR}/arch/sparc/src/sparc_v8/Toolchain.defs
 
 LDSCRIPT = linksparc.ld
 
-ifeq ($(WINTOOL),y)
-  # Windows-native toolchains
-  DIRLINK = $(TOPDIR)/tools/copydir.sh
-  DIRUNLINK = $(TOPDIR)/tools/unlink.sh
-  MKDEP = $(TOPDIR)/tools/mkwindeps.sh
```

Comment:
Why `DIRLINK`, `DIRUNLINK` and `MKDEP` are removed?

Reply:
Defined in common place now:
https://github.com/apache/incubator-nuttx/blob/master/tools/Config.mk#L164-L190